changelog.txt

```diff
@@ -1,6 +1,9 @@
 VNext
 ----------
 
+Version 3.4.5
+----------
+
```

Comment:
actual changelog?

Reply:
Not in this PR because there are no changes pulled into this release yet